django_app/redbox_app/redbox_core/models.py

```diff
@@ -64,6 +69,7 @@ class File(UUIDPrimaryKeyBase, TimeStampedModel):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     original_file_name = models.TextField(max_length=2048, blank=True, null=True)
     core_file_uuid = models.UUIDField(null=True)
+    expiry_date = models.DateField(default=get_default_expiry_date)
```

Comment:
either:
* this needs to be nullable or we will get an error on migration, or
* we could compute this field on the fly from the `created_at: datetime`
?